web/source/dom/domManager.ts

```diff
@@ -304,22 +317,24 @@ namespace com.keyman.dom {
      *              Note that the 'kmw-disabled' property is managed by the MutationObserver and by the surface API calls.
      */
     enableInputElement(Pelem: HTMLElement, isAlias?: boolean) {
```

Comment:
```suggestion
    enableInputElement(Pelem: HTMLElement) {
```

Shouldn't the `isAlias` parameter be deleted? See also line 247 above.